src/routes/authors.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const authorsController = require('../controllers/authors');
+
+const router = express.Router();
+
+router.post('/', authorsController.createAuthor);
+
+router.get('/', authorsController.getAllAuthors);
+router.get('/:id', authorsController.getAuthorById);
+
+router.patch('/:id', authorsController.updateAuthor);
+
+router.delete('/:id', authorsController.deleteAuthor);
```

Comment:
I see what you are going for here. In this case as there are only a handful of routes it would be okay to bunch them up together as it wouldnt affect the readability of it
```
router.post('/', authorsController.createAuthor);
router.get('/', authorsController.getAllAuthors);
router.get('/:id', authorsController.getAuthorById);
router.patch('/:id', authorsController.updateAuthor);
router.delete('/:id', authorsController.deleteAuthor);
```